### DISCUSSION TITLE
Icons Not Working!

### DISCUSSION THREAD

**Discussion by mosespace** [fuma-nama/fumadocs#1149] [Q&A]
Hy, ive tried to create a new fuma-docs app following the documentation, but the icons are not working as showin below 
![image](https://github.com/user-attachments/assets/27a10772-ef42-4051-991b-0a73871df588)
![image](https://github.com/user-attachments/assets/91334989-39c6-45f8-aa30-6c7276fde4bb) and also how can i customze this to look like the one on the docs please 
![image](https://github.com/user-attachments/assets/27353b49-23be-439d-833f-dcb0a7c9ca37)

**Comment by fuma-nama** [comment]
https://fumadocs.vercel.app/docs/headless/source-api#icons

**Reply by mosespace** [reply]
I tried that but didn't work for me!
Like i don't know were to put it!!!

**Reply by mosespace** [reply]
Thank you after fully trying to understand the docs, i had to just update my sourse.ts in the libs.

**Comment by mosespace** [comment]
am now wondering how to customize this 
![image](https://github.com/user-attachments/assets/ac659346-bd83-483c-bd8e-0c8ddb100d4b)

**Reply by fuma-nama** [reply]
https://fumadocs.vercel.app/docs/ui/blocks/page#style

**Comment by mosespace** [comment]
Thank you, it worked for me 🙏
